test/specs/configs/urls.json

```diff
@@ -92,11 +94,9 @@
         "repo": "project-helix.io"
       },
       "sticky": false,
-      "url": "https://client.project-helix.io/",
       "urls": [
         "https://client.project-helix.io/index.html",
-        "https://client.project-helix.io/README.html",
-        "https://client.project-helix.io/"
```

Comment:
why did you remove those?

Reply:
I just removed the last line (`"https://client.project-helix.io/"`) as it was inferred by the `url` property value that was removed.